File: C/Reverse.c
#include<stdio.h>
void main()
{
	int num,r,a=0,i;
	printf("Enter a number to find the reverse \t");
	scanf("%d",&num);
	i=num;
	while(num>0)
	{
		r=num%10;
		a=a*10+r;
		num=num/10;
	}
	printf("The reverse of number %d is %d \n",i,a);
}

File: JAVA/Reverse.java
import java.io.*;
class Reverse
{
	public static void main(String args[])
	{
		int i,num=513,s=0,a;
		i=num;
		while(num>0)
		{
			a=num%10;
			s=s*10+a;
			num=num/10;
		}
		System.out.println("Sum of digits of "+i+" is: "+s);
	}
}
